### PR TITLE
add igor package

### PR DESCRIPTION
**Pull request by carlodri**

### CODE REVIEW

File: recipes/igor/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - matplotlib
+
+test:
+  imports:
+    - igor
```
Comment (synapticarbors) [inline recipes/igor/meta.yaml]: add an import test for `igor.record`

Also, should the two scripts be tested? 

```
'bin/igorbinarywave.py',
'bin/igorpackedexperiment.py'
```

Comment (carlodri) [reply recipes/igor/meta.yaml]: thanks @synapticarbors for the review! I have looked into the package structure and updated the import and command tests to what seems the best to me. Feel free to comment back if you don't agree with my choices.

**Comment by synapticarbors** [comment]
I'm not entirely sure what's going on with the permission error from the scripts. Not sure if it might might an issue with the scripts not being executable and being called just as `script.py`.

**Comment by carlodri** [comment]
Yes, I was thinking the same thing and I am looking into a solution. Perhaps I would simply call the scripts as `python scripy.py`.

**Comment by carlodri** [comment]
looks like it's probably working now! 🎉 

That said, the proper solution AFAIU would be to `chmod +x` the two scripts _in_ the source repo...

**Pull request merged**